sdk/src/bigint.rs

```diff
@@ -125,10 +125,11 @@ impl TryFrom<BigInt> for ScVal {
 impl TryFrom<EnvType<ScVal>> for BigInt {
     type Error = ConversionError;
     fn try_from(v: EnvType<ScVal>) -> Result<Self, Self::Error> {
-        v.val
+        let ev: EnvObj = v
+            .val
             .try_into_env_val(&v.env)
-            .map_err(|_| ConversionError)?
-            .try_into()
+            .map_err(|_| ConversionError)?;
+        ev.try_into()
```

Comment:
These type annotations are necessary now per the comment here: https://github.com/stellar/rs-stellar-contract-env/pull/252#issue-1315554643.